package-lock.json

```diff
@@ -45,6 +45,7 @@
         "pino-pretty": "^10.0.0",
         "rimraf": "^5.0.0",
         "supertest": "^6.3.3",
+        "swagger-spec-validator": "^5.0.0",
```

Comment:
Thanks for the warning regarding the online server use. I thought I would recommend [redocly](https://www.npmjs.com/package/@redocly/cli) (from [openapi.tools](https://openapi.tools/#description-validators)) but when I ran it it gave like 25 more errors on this branch. I suppose if those are substantial (i.e. a generator won't work) then we should switch. In any case, this is certainly an improvement.

Reply:
Absolutely -- I'll open an issue to capture this as well!